rcsb/utils/targets/CARDTargetAnnotationProvider.py

```diff
@@ -0,0 +1,213 @@
+##
+#  File:           CARDTargetAnnotationProvider.py
+#  Date:           6-Mar-2023 dwp
+#
+#  Updated:
+#
+##
+"""
+Accessors and generators for CARD target annotation data.
+"""
+
+import datetime
+import logging
+import os.path
+import time
+
+from rcsb.utils.io.FileUtil import FileUtil
+from rcsb.utils.io.MarshalUtil import MarshalUtil
+from rcsb.utils.io.StashableBase import StashableBase
+from rcsb.utils.targets.CARDTargetProvider import CARDTargetProvider
+
+logger = logging.getLogger(__name__)
+
+
+class CARDTargetAnnotationProvider(StashableBase):
+    """Accessors and generators for CARD target annotation data."""
+
+    def __init__(self, **kwargs):
+        #
+        self.__cachePath = kwargs.get("cachePath", ".")
+        self.__dirName = "CARD-annotations"
```

Comment:
This will be a new file in the BuildLocker stash instead of `CARD-features`, is that correct? If so, we'll need to run the workflow to update the file.